src/modules/flight_mode_manager/tasks/Orbit/FlightTaskOrbit.cpp

```diff
@@ -143,11 +136,12 @@ bool FlightTaskOrbit::sendTelemetry()
 	return true;
 }
 
-void FlightTaskOrbit::_sanitizeParams(float &radius, float &velocity) const
+void FlightTaskOrbit::_sanitizeParams(float &radius, float &velocity)
```

Comment:
Why not leave this as const? We are not modifying any member variables

Reply:
Good catch. I used to update parameters in this function, but now that it is moved out, it should be const.